docs/rss-tutorials/state_estimation_tutorial_preparations.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: State Estimation Tutorial
+parent: Summer School Tutorials
+---
+
+# Preparations for State Estimation Tutorial
+For the tutorial you need the following packages:
+ - [ethzasl_msf](https://github.com/ethz-asl/ethzasl_msf)
+ - [compslam](https://bitbucket.org/leggedrobotics/compslam_rss/src/master/)
+
+You can test it on your own computer by running it with a rosbag provided [bellow](#Download files).
+
+## Installation
+Download the repositories and build them:
+
+```bash
+cd ~/catkin_ws/src/
+git clone https://github.com/ethz-asl/ethzasl_msf.git
+git clone https://github.com/ethz-asl/glog_catkin.git
+git clone https://github.com/catkin/catkin_simple.git
+catkin build loam
+catkin build ethzasl_msf
+catkin build smb_msf
+source ~/<catkin_ws>/devel/setup.bashrc
```

Comment:
please be consistent when specifying a path: use placeholders everywhere or nowhere.

Reply:
Thanks for catching that!